.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_SERVER_URL=http://localhost:8080/termit
+REACT_APP_DEPLOYMENT_NAME=
```

Comment:
Consider adding a newline
https://mrkiffie.com/2016/newlines-and-version-control/